### ISSUE TITLE
Multiple peer projects

### ISSUE THREAD

**Issue by devans** [olim7t/sbt-scalariform#1]
In the case where you have a parent project that's managing multiple sub projects (which it's instantiating), the scalariform jar seems to get put into the lib_managed/ directory of whatever the first ScalariformPlugin project is, and other projects can't find it.

**Comment by olim7t** [comment]
Is this happening with the 1.0.0 release? I've changed how the dependency to Scalariform is managed, but this didn't get into a release yet. Could you try with the latest head version? The idea is that Scalariform is now a dependency of the plugin, that is declared on the parent project; so the jar goes to its `./project/plugins/lib_managed/scala_xxx/` directory, and a child project that mixes in `ScalariformPlugin` will now also look for the jar in its parent's dependencies.
It that solves the issue, I'll release a 1.0.1 shortly.
If not, could you provide a test case that reproduces the issue?

**Comment by devans** [comment]
I'd like to try it out, but I can't get scalariform to build (I assume it's something to do with windows), and I can't figure out how to get the plugin to refer to and older version.

**Comment by olim7t** [comment]
I meant the latest head version of sbt-scalariform:

```
git clone git://github.com/olim7t/sbt-scalariform.git
cd sbt-scalariform
sbt publish-local
```

And then change your project to depend on version 1.0.1-SNAPSHOT of the plugin.

**Comment by devans** [comment]
yeah, it asks for 0.5-SNAPSHOT of scalariform, I can't figure out what a previous version of scalariform is that I can set it to (it uses some weird number string like 2071832) , and I can't get the scalariform head to build.

**Comment by olim7t** [comment]
Scalariform 0.0.5-SNAPSHOT is published at the scala-tools repository, it should grab it from there. Try running `update` before `publish-local`.
I'll be back home shortly, I'll try to clean my local caches and rebuild from scratch.

**Comment by devans** [comment]
This is what I get when I update my project:
[warn]  ==== Scala-Tools Maven2 Repository: tried
[warn]    http://scala-tools.org/repo-releases/org/scalariform/scalariform_2.8.0/0.0.5-SNAPSHOT/scalariform_2.8.0-0.0.5-SNAPSHOT.pom
[warn]    -- artifact org.scalariform#scalariform_2.8.0;0.0.5-SNAPSHOT!scalariform_2.8.0.jar:
[warn]    http://scala-tools.org/repo-releases/org/scalariform/scalariform_2.8.0/0.0.5-SNAPSHOT/scalariform_2.8.0-0.0.5-SNAPSHOT.jar

**Comment by olim7t** [comment]
Was solved by building 1.0.1-SNAPSHOT locally.
Released 1.0.1 with explanations for multi-module projects, which will hopefully make things easier.
